interp/interp_test.go

```diff
@@ -68,21 +70,18 @@ func BenchmarkFindSegment(b *testing.B) {
 // testPiecewiseInterpolatorCreation tests common functionality in creating piecewise  interpolators.
 func testPiecewiseInterpolatorCreation(t *testing.T, fp FittablePredictor) {
 	type errorParams struct {
-		xs              []float64
-		ys              []float64
-		expectedMessage string
+		xs []float64
+		ys []float64
 	}
 	errorParamSets := []errorParams{
-		{[]float64{0, 1, 2}, []float64{-0.5, 1.5}, "xs and ys have different lengths"},
-		{[]float64{0.3}, []float64{0}, "too few points for interpolation"},
-		{[]float64{0.3, 0.3}, []float64{0, 0}, "xs values not strictly increasing"},
-		{[]float64{0.3, -0.3}, []float64{0, 0}, "xs values not strictly increasing"},
+		{[]float64{0, 1, 2}, []float64{-0.5, 1.5}},
+		{[]float64{0.3}, []float64{0}},
+		{[]float64{0.3, 0.3}, []float64{0, 0}},
+		{[]float64{0.3, -0.3}, []float64{0, 0}},
 	}
 	for _, params := range errorParamSets {
-		err := fp.Fit(params.xs, params.ys)
-		expectedMessage := fmt.Sprintf("interp: %s", params.expectedMessage)
-		if err == nil || err.Error() != expectedMessage {
-			t.Errorf("expected error for xs: %v and ys: %v with message: %s", params.xs, params.ys, expectedMessage)
+		if !panics(func() { fp.Fit(params.xs, params.ys) }) {
```

Comment:
`if !panics(func() { _ = fp.Fit(params.xs, params.ys) }) {`

Reply:
It seems to work without the underscore, so why make the code longer?

Reply:
To keep the linter quiet.

Reply:
It leaves annotations in the files changed tab.